szachy.js

```diff
@@ -11,13 +11,19 @@ document.addEventListener("DOMContentLoaded", function onDOMLoaded() {
     *  -> chessline (of chessboard)
     *      for
     *      -> field (of chessline) */
-    for (let i = 0; i < 64; i++) {
-        let pole = document.createElement("div");
-        pole.innerText = "P:" + i;
-        pole.classList.add("field");
-        chessboard.appendChild(pole);
-        console.log(`${i} pole: `, pole);
+    
+        for (let j=8 ;j>0; j--){
+            let chessline = document.createElement("div")
+            chessline.classList.add("chessline");
+            chessboard.appendChild(chessline); 
+            for (let i = 0; i <8 ; i++) {
+                let pole = document.createElement("div");
+                pole.innerText = "P" + j;
+                pole.classList.add("field");
+                chessline.appendChild(pole);
+                console.log(`${i} pole: `, pole);
```

Comment:
That's good, but still need's id's for each of the fields to be implemented.
setting innerText could be scrapped.